### DISCUSSION TITLE
Disabling Mouse Click for native tablet drivers on osu!lazer

### DISCUSSION THREAD

**Discussion by AkinariHex** [ppy/osu#19439] [Ideas] [answered]
The tip of my tablet pen is broken and always get the input for mouse click (the pen tip is stuck inside so it gets always the mouse click), with TabletDriver program I disabled the pen tip clicks and I would like to be able to do the same with tablet drivers that are natively on osu!lazer. (I've seen another discussion about this but seems no button is here to disable the click)

I've already checked the disable mouse click while playing but the problem still in the menu.
In the video below I was hovering with the pen

https://user-images.githubusercontent.com/28952344/181451675-2ed885c2-7d80-4c59-b3d1-6a9e9355b215.mp4

Another suggestion is the possibility to keybind with pen buttons, personally I use the pen buttons to restart a map but when trying to change a keybind the pen buttons are not recognized

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/14037
